test/Removal.t.sol

```diff
@@ -456,12 +504,15 @@ contract Removal__beforeTokenTransfer is NonUpgradableRemoval {
     );
   }
 
-  function test_zeroValueTransfer_reverts_RemovalAmountZero() external {
+  function test_zeroValueTransferToMarket_reverts_RemovalAmountZero() external {
+    address mockMarketAddress = vm.addr(1);
+    // todo figure out stdstorage
+    // stdstore.target(address(super)).sig(super._market.selector).checked_write(mockMarketAddress);
```

Comment:
this is currently broken because i changed the implementation such that transfers with 0 amounts are only reverted if they are to or from the market contract (a decision we could revisit, it just seemed more efficient bc it shouldn't otherwise matter).  so I'm trying to get this test to work by giving the test contract a notion of what the market contract's address is, which is not straightforward but *should* be possible without over-complicating this test with a full upgradeablemarket ecosystem

Reply:
resolved